### PR TITLE
Updated the update function to fix issue #176

### LINKED ISSUE #150: Update command does not separate arguments from switches
Update command attempts to format the switch and argument as a number, resulting in unhandled NumberFormatExceptions.

<hr><sub>[original: aquohn/ped#14]<br/>
</sub>

### LINKED ISSUE #151: Update command allows empty updates
If the update command is entered without arguments, it is not rejected even though there is no meaning to the command. A fatal NumberFormatException will be thrown.

<hr><sub>[original: aquohn/ped#15]<br/>
</sub>

### LINKED ISSUE #165: Update function doesn't seem to take in input.
Given input "Y" but response seem to be "stopped the update". "N" gives the same output as well.

Exception thrown when I tried the same command with "y"

<hr><sub>[original: Greatnest/ped#7]<br/>
</sub>

### LINKED ISSUE #176: Update tags not checked
When a tag with no value is given, prompt is continued.

As update is not done, unsure of the effects.

<hr><sub>[original: Greatnest/ped#18]<br/>
</sub>

### LINKED ISSUE #188: 'update' feature does not seem to be working
<hr><sub>[original: aarushisingh1/ped#10]<br/>
</sub>

### PR DESCRIPTION
**Pull request by koushireo**
Fixed issue. closes #176 closes #188 closes #165 closes #151 closes #150

### CODE REVIEW

File: src/main/java/duke/logic/commands/UserSetup.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public void initialise(String info) {
             }
             setGender(sex);
         } else if (user.getActivityLevel() == 5) {
-            int activity = 0;
+            int activity = 5;
```
Comment (HashirZahir) [inline src/main/java/duke/logic/commands/UserSetup.java]: Why was this changed?

File: src/main/java/duke/logic/commands/UpdateWeightCommand.java [hunk 2]
```diff
@@ -71,18 +71,23 @@ public void stage0(User user, Storage storage) {
                 ui.showWeightUpdate(user, Integer.parseInt(weight), currentDate);
                 ui.showLine();
             } catch (NumberFormatException e) {
-                ui.showMessage(e.getMessage());
+                ui.showMessage("Please input a proper number for weight");
```
Comment (HashirZahir) [inline src/main/java/duke/logic/commands/UpdateWeightCommand.java]: Good job adding exception handling. Perhaps you could be more specific with the error message? Proper number may not be clear for new users. Maybe something like: "<user input> is not a valid weight. Please input your weight as a positive number". Also, do you check for negative weight inputs?

File: src/main/java/duke/ui/Ui.java [hunk 3]
```diff
@@ -293,9 +293,7 @@ public void showInsufficientBalance(Payment payment) {
     }
 
     public void showSuccess(String type, String value) {
-        showLine();
         System.out.println(UI_PADDING + "Your " + type + " has been updated to " + value + ".");
```
Comment (HashirZahir) [inline src/main/java/duke/ui/Ui.java]: Can we standardize all calls to ` System.out.println(UI_PADDING + message);` in Ui.java to be replaced with `showMessage(message)`

File: src/main/java/duke/logic/commands/UserSetup.java (hunk 1 above)
Comment (koushireo) [reply src/main/java/duke/logic/commands/UserSetup.java]: activity level ranges from 0 to 4 in the program while it's displayed at 1-5 in UI output. therefore putting an impossible value can help to check for validation as int cannot be null

**Pull request merged**